services/supervisor/src/proto_structures.rs

```diff
@@ -20,14 +20,15 @@ use exonum::{
     exonum_merkledb::ObjectHash,
```

Comment:
Off-topic: Huh, does the main crate expose both `merkledb` and `exonum_merkledb`? Seems weird.

Reply:
I already grumbled about that. If I remember correctly, this is due to some issues with derive macros. And I totally agree that it's way too weird =_=